unlock-protocol.com/blog/portability.md

```diff
@@ -0,0 +1,21 @@
+---
+title: 'What I’m Excited About: Portability'
+subTitle: Instead of ads following us around our access should.
+authorName: Sascha Mombartz
+publishDate: Feb 19, 2020
+description: Instead of ads following us around our access should.
+image: /static/images/blog/portability/hero.jpg
+---
+
+![Take it with you](/static/images/blog/portability/hero.jpg)
+
+
+You’ve experienced this: You‘ve bought something (or just searched for it) and now you have ads for that or a similar item following you around. What’s happening is you have two services – the shop you bought from and the ad delivery network – that are talking to you but not each other, and that leads to a not so great user experience. 
```

Comment:
```suggestion
You’ve experienced this: you‘ve bought something (or just searched for it) and now you have ads for that or a similar item following you around. What’s happening is you have two services—the shop you bought from and the ad delivery network—that are talking to you but not each other, and that leads to a not-so-great user experience. 
```

Reply:
I changed the hyphens after "two services" to be em dashes, but not sure if they display correctly here

Reply:
I hate em dashes and only use en dashes! but thats ok.